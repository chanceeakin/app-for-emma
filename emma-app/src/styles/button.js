// @flow
import colors from './colors'
import type Styles from './../types/Styles.js.flow'

const button: Styles = {
  bigButton: {
    marginTop: 20,
    paddingVertical: 15,
    paddingHorizontal: 40,
    borderStyle: 'solid',
    borderWidth: 1,
    borderColor: colors.inkBlue,
    borderRadius: 5,
  },
  bigButtonText: {
    fontSize: 23,
    fontWeight: '600',
    color: colors.inkBlue,
  },
  settingsButton: {
    marginTop: 20,
    paddingVertical: 10,
    paddingHorizontal: 30,
    backgroundColor: colors.duskyRose,
    borderRadius: 5,
  },
  settingsButtonText: {
    fontSize: 16,
    fontWeight: '600',
    color: colors.white,
  },
  mediumButton: {
    marginTop: 20,
    paddingVertical: 10,
    paddingHorizontal: 30,
    backgroundColor: colors.darkBlue,
    borderRadius: 5,
  },
  mediumButtonText: {
    fontSize: 16,
    fontWeight: '600',
    color: colors.white,
  },
  errorButton: {
    backgroundColor: colors.error,
  },
  errorText: {
    color: colors.white,
  },
  backButton: {
    padding: 15,
  },
}

export default button
