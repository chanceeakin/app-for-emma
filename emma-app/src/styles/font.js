import type Styles from './../types/Styles.js.flow'

export const font: Styles = {
  skia: {
    fontFamily: 'Skia'
  }
}

export default font
