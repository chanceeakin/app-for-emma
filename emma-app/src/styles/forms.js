import colors from './colors'

export const forms: Object = {
  input: {
    height: 40,
    borderColor: 'gray',
    borderBottomWidth: 1,
    color: colors.darkBlue,
    minWidth: 200
  }
}

export default forms
