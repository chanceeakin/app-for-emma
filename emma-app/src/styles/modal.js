import type Styles from './../types/Styles.js.flow'

export const modal: Styles = {
  modal: {
    flex: 1,
    paddingTop: 30,
    justifyContent: 'space-between',
    alignItems: 'center'
  },
  modalTitle: {
    fontSize: 45,
    textAlign: 'center'
  }
}

export default modal
