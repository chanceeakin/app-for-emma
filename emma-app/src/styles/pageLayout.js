export const pageLayout = {
  container: {
    flex: 1,
    justifyContent: 'space-between',
    alignItems: 'center'
  }
}

export default pageLayout
