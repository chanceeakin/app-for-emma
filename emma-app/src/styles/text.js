// @flow
import type Styles from './../types/Styles.js.flow'
import colors from './colors'

const text: Styles = {
  title: {
    fontSize: 72,
    textAlign: 'center',
    color: colors.inkBlue,
  },
  description: {
    fontSize: 30,
    textAlign: 'center',
    color: colors.darkPurple,
  },
}

export default text
