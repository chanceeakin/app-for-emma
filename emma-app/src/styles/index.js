import button from './button'
import modal from './modal'
import colors from './colors'

export { button, modal, colors }

export default { button, modal, colors }
