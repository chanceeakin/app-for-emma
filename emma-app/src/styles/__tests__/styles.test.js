import React from 'react'
import styles from '../index'

describe('Styles', () => {
  it('renders correctly', () => {
    expect(styles).toMatchSnapshot()
  })
})
