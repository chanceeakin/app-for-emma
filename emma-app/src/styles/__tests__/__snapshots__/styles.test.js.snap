// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles renders correctly 1`] = `
Object {
  "button": Object {
    "backButton": Object {
      "padding": 15,
    },
    "bigButton": Object {
      "borderColor": "#0D1B2A",
      "borderRadius": 5,
      "borderStyle": "solid",
      "borderWidth": 1,
      "marginTop": 20,
      "paddingHorizontal": 40,
      "paddingVertical": 15,
    },
    "bigButtonText": Object {
      "color": "#0D1B2A",
      "fontSize": 23,
      "fontWeight": "600",
    },
    "errorButton": Object {
      "backgroundColor": "#CC2936",
    },
    "errorText": Object {
      "color": "#fafafa",
    },
    "mediumButton": Object {
      "backgroundColor": "#114B5F",
      "borderRadius": 5,
      "marginTop": 20,
      "paddingHorizontal": 30,
      "paddingVertical": 10,
    },
    "mediumButtonText": Object {
      "color": "#fafafa",
      "fontSize": 16,
      "fontWeight": "600",
    },
    "settingsButton": Object {
      "backgroundColor": "#D37482",
      "borderRadius": 5,
      "marginTop": 20,
      "paddingHorizontal": 30,
      "paddingVertical": 10,
    },
    "settingsButtonText": Object {
      "color": "#fafafa",
      "fontSize": 16,
      "fontWeight": "600",
    },
  },
  "colors": Object {
    "background": "#f9f9f9",
    "darkBlue": "#114B5F",
    "darkPurple": "#4C3B4D",
    "duskyRose": "#D37482",
    "error": "#CC2936",
    "green": "#89BD9E",
    "inkBlue": "#0D1B2A",
    "lightGreen": "#E5FCF5",
    "lightOrange": "#EFC7C2",
    "lightPeach": "#FFE5D4",
    "slateBlue": "#3A5760",
    "white": "#fafafa",
  },
  "font": Object {
    "skia": Object {
      "fontFamily": "Skia",
    },
  },
  "forms": Object {
    "input": Object {
      "borderBottomWidth": 1,
      "borderColor": "gray",
      "color": "#114B5F",
      "height": 40,
      "minWidth": 200,
    },
  },
  "modal": Object {
    "modal": Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "space-between",
      "paddingTop": 30,
    },
    "modalTitle": Object {
      "fontSize": 45,
      "textAlign": "center",
    },
  },
  "pageLayout": Object {
    "container": Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "space-between",
    },
    "row": Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "minHeight": 100,
    },
  },
  "text": Object {
    "description": Object {
      "color": "#4C3B4D",
      "fontSize": 30,
      "textAlign": "center",
    },
    "title": Object {
      "color": "#0D1B2A",
      "fontSize": 72,
      "textAlign": "center",
    },
  },
}
`;
